### ISSUE TITLE
The lexicon terms shown as a strange characters

### ISSUE THREAD

**Issue by yemen2016** [nora-twairesh/AraSenti#3]
Dear Madam,

Could you please help me how to get the lexicon in Arabic characters. I am just find all the terms as a strange characters.

**Comment by abeermohamed1** [comment]
yes if you open it on GitHub, so download then try to open with "notebad++" will work fine

**Comment by yemen2016** [comment]
I got the same issue even after open it using notepad++ or emeditor.
Could please fix this issue?

**Comment by RamiShahazeh** [comment]
I even tried couple of encoding on Notepad++,  it doesn't work ... Can you please upload a CSV version of it.

**Comment by Reem-mk** [comment]
me too , I faced the same problem even after open it using notepad++ or emeditor.

**Comment by areejokaili** [comment]
Hi
If anyone is interested, I've read the data and stored it in a CSV file  [AraSentiLexiconV1.0.csv](https://github.com/nora-twairesh/AraSenti/files/7751536/AraSentiLexiconV1.0.csv).


Read it as follows (in Python):
`import pandas as pd`
`data= pd.read_csv('AraSentiLexiconV1.0.csv')`

Or read the original data text file but make sure to set the encoding to `windows-1256`.


Hope this is useful.
